src/utils/walletConfig.ts

```diff
@@ -0,0 +1,194 @@
+import {
+  AcceptedToken,
+  getChainId,
+  getTokenAddress,
+  SupportedChain,
+} from '@/types/chains'
+
+export interface Currency {
+  name: string
+  code: string
+  flag: string
+}
+
+export interface Network {
+  name: string
+  icon: string
+  chainId: number
+}
+
+export interface CryptoAsset {
+  name: string
+  symbol: string
+  icon: string
+  price: string
+  balance: string
+  usdValue: string
+  fullBalance?: string
+  currencyIcon?: string
+  tokenAddress: string
+  chainId: number
+  networkName?: string
+  isLoading?: boolean
+}
+
+export interface CryptoConfig {
+  name: string
+  symbol: string
+  icon: string
+  price: string
+  tokenAddress: string
+  celoChainId: number
+  arbitrumTokenAddress?: string
+  arbitrumChainId?: number
+  arbitrumSepoliaTokenAddress?: string
+  arbitrumSepoliaChainId?: number
+}
+
+// Centralized currency configuration
+export const CURRENCIES: Currency[] = [
+  { name: 'US Dollar', code: 'USD', flag: '/assets/currencies/usd.png' },
+  { name: 'Euro', code: 'EUR', flag: '/assets/currencies/euro.png' },
+  {
+    name: 'Pounds sterling',
+    code: 'GBP',
+    flag: '/assets/currencies/pounds.png',
+  },
+]
+
+// Centralized network configuration using chains.ts
+export const NETWORKS: Network[] = [
```

Comment:
This should be based on the environment, I.e, supportedPaymentChains on the constant/meeting-types file